crates/uplc/src/optimize/shrinker.rs

```diff
@@ -226,10 +228,13 @@ impl DefaultFunction {
                     false
                 }
             }),
-            DefaultFunction::EqualsByteString
+            DefaultFunction::LengthOfByteString
+            | DefaultFunction::EqualsByteString
             | DefaultFunction::AppendByteString
             | DefaultFunction::LessThanEqualsByteString
-            | DefaultFunction::LessThanByteString => arg_stack.iter().all(|arg| {
+            | DefaultFunction::LessThanByteString
+            | DefaultFunction::DecodeUtf8
```

Comment:
I don't believe this is generally true? Decoding an invalid bytestring will throw / error. Though maybe I am misunderstanding this function (does it only look at arguments?)

Reply:
Oh you are right. I'll remove it.